pywbem/config.py

```diff
@@ -93,3 +93,17 @@
 #:
 #: *New in pywbem 0.12.*
 SEND_VALUE_NULL = True
+
+#: Backward compatibility option that controls creating CIMInstanceNames with
+#: null values for keybindings.
+
+#: :term:`DSP0004`, section 7.9 specifically forbids key properties with
+#: no values but because pywbem has always allowed this, adding the code to
+#: disallow None as a keybinding value is an incompatible change.
+#:
+#: * True: Pywbem ignores None as a value when keybindings are defined.
+#: * False (default): Pywbem generates ValueError when keybindings are created
```

Comment:
It does not "generate" ValueError, but "raises" ValueError. Also, use a link for ValueError.